redbot/cogs/economy/economy.py

```diff
@@ -922,16 +927,19 @@ async def slottime(self, ctx: commands.Context, seconds: int):
         await ctx.send(_("Cooldown is now {num} seconds.").format(num=seconds))
 
     @economyset.command()
-    async def paydaytime(self, ctx: commands.Context, seconds: int):
+    async def paydaytime(self, ctx: commands.Context, seconds: TimedeltaConverter):
```

Comment:
It probably makes sense to set default unit to seconds here:
```suggestion
    async def paydaytime(
        self, ctx: commands.Context, seconds: TimedeltaConverter(default_unit="seconds")
    ):
```